srcpkgs/comfy/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'comfy'
+pkgname=comfy
+version=0.5.4
+revision=1
+build_style=cargo
+short_desc="Comfy is a flexible command script manager / runner written in Rust"
```

Comment:
Drop the "Comfy is a" and just start it like "Flexible command..."

Reply:
Done. Thanks for the tip.